lib/Doctrine/DBAL/Migrations/Tools/Console/Command/MigrateCommand.php

```diff
@@ -144,48 +146,58 @@ public function execute(InputInterface $input, OutputInterface $output)
 
             if (empty($sql)) {
                 $output->writeln('<comment>No migrations to execute.</comment>');
+                $exitCode = ($input->getOption('allow-no-migration')) ? 0 : 1;
+
+                return $exitCode;
             }
         }
+
+        return 1;
     }
 
     /**
-     * @param string $question
-     * @param InputInterface $input
+     * @param string          $question
```

Comment:
@mikeSimonson : And waht about phpdoc ? Should I keep it that way or rollback to the original one ?